### PR TITLE
Register vrbz.is-a.dev

### PR DESCRIPTION
**Pull request by vrbz**
Register vrbz.is-a.dev with CNAME record pointing to 3e898fe9847779bdcaea77cb3b4fcef5.is-a.dev.

### CODE REVIEW

**Comment by vrbz** [comment]
also, im not entirely sure if thats the correct thing, so just in case, the website is vrbz.rf.gd

File: domains/vrbz.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "vrbz",
+           "email": "108748279+vrbz@users.noreply.github.com",
+           "discord": "590921984020447235"
+        },
+    
+        "record": {
+            "CNAME": "3e898fe9847779bdcaea77cb3b4fcef5.is-a.dev"
```
Comment (CuteDog5695) [inline domains/vrbz.json:9]: 3e898fe9847779bdcaea77cb3b4fcef5.is-a.dev ofc doesnt exist, please change it to an actual valid CNAME record

File: domains/vrbz.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "vrbz",
+           "email": "108748279+vrbz@users.noreply.github.com",
+           "discord": "590921984020447235"
```
Comment (CuteDog5695) [inline domains/vrbz.json:5]: either replace email line with a email which can be contacted or remove the github no reply email

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/vrbz.json (hunk 1 above)
Comment (CuteDog5695) [reply domains/vrbz.json:9]: > if i attempt to add the subdomain it tells me i first need to link the actual domain to be able to add subdomains, so im not sure if its possible to use with that service

if its asking you to connect is-a.dev and then vrbz.is-a.dev then yea it isnt possible, if you want to you can try github pages or cloudflare pages

**Pull request closed without merging**